language/move-lang/tests/move_check/expansion/duplicate_function_in_module.exp

```diff
@@ -3,9 +3,9 @@ error:
    ┌── tests/move_check/expansion/duplicate_function_in_module.move:3:9 ───
    │
  3 │     fun foo() { }
-   │         ^^^ Duplicate definition for function 'foo' in module 'M'
+   │         ^^^ Duplicate module member or alias 'foo'. Top level names in a module must be unique
    ·
  2 │     fun foo() { }
-   │         --- Previously defined here
+   │         --- Previously declared here
```

Comment:
I don't have a strong opinion here, but I'm curious about how you're thinking about definitions vs. declarations. Those have particular meanings for C-based languages, but I don't know if those carry over to many other languages. In C terms, this is a definition.

Reply:
I'll switch to defined. But we should be cautious of this. What is the difference in C?

Reply:
https://stackoverflow.com/questions/1410563/what-is-the-difference-between-a-definition-and-a-declaration

Move doesn't distinguish these -- you can't declare a function without also defining it -- so I think we could use either term. It would be good to be consistent and I was curious why you changed this.

Reply:
It changed because I was inconsistent and now a different place caught the error :p

Reply:
I think "declare" might make ever-so-slightly more sense because of native functions (which  are declared in Move, but defined in the host language).